site/_scss/blocks/_project-sections.scss

```diff
@@ -18,6 +18,7 @@
 
   @include media-query('lg') {
     flex-direction: row;
+    gap: 0 px-to-rem(56px);
```

Comment:
FYI: This was added because long titles in the left-hand column were overlapping with links in the right-hand column.